src/EditorFeatures/Test/EditAndContinue/EditSessionActiveStatementsTests.cs

```diff
@@ -6,20 +6,21 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.EditAndContinue;
-using Microsoft.CodeAnalysis.EditAndContinue.UnitTests;
+using Microsoft.CodeAnalysis.CSharp.EditAndContinue;
```

Comment:
This file has been moved from CSharpTest to general EnC tests. If there is still a dependency to CSharp, it maybe reasonable to keep the test in the CSharp project and namespace.

Reply:
The functionality being tested is not specific to C#. Just need some language...